.github/workflows/ci.yml

```diff
@@ -44,6 +44,16 @@ jobs:
           ./mconfig -v -p /usr/local
           make -C ./builddir check
 
+  shellcheck:
+    name: shellcheck
+    runs-on: ubuntu-20.04
+    container: koalaman/shellcheck-alpine
+    steps:
+      - uses: actions/checkout@v2
+      - name: shellcheck files that appear to be sh or bash scripts (or some cousin thereof)
+        run: |
+          shellcheck $(grep -rm1 ^ * | grep -v "^vendor|^builddir" | grep '#!/bin/*.sh' | sed -e 's|:#!.*||') || true
```

Comment:
I think that's supposed to be `'#!/bin/.*sh'`.  As it is, it is only matching `#!/bin/sh` and not any other shells.  The only reason it is matching that is because the `/*` is treated as zero characters and the `.` matches the `/`.

Reply:
I should have marked it WIP -- caught that one and replaced the complex command with a shell script